File: java/ofdpdfsigner_jni/src/main/java/com/fzz/seal/entity/KeyPair.java
package com.fzz.seal.entity;

public class KeyPair {
    private com.fzz.seal.entity.Key pubKey;

    private com.fzz.seal.entity.Key prvKey;




    public KeyPair(com.fzz.seal.entity.Key pubKey, com.fzz.seal.entity.Key prvKey) {
        this.pubKey = pubKey;
        this.prvKey = prvKey;
    }


    public com.fzz.seal.entity.Key getPublicKey() {
        return pubKey;
    }


    public com.fzz.seal.entity.Key getPrivateKey() {
        return prvKey;
    }
}

File: ios/ofdpdfsigner_ios/ofdpdfsignerlib/util/KeyPair.swift
//
//  KeyPair.swift
//  ofdcreatorlib
//
//  Created by zizheng fan on 2024/5/31.
//

import Foundation

public class KeyPair {
    
    private var pubKey: Key?
    private var prvKey: Key?
  
    init() {
        
    }
  
    public func getPublicKey() -> Key? {
        return pubKey
    }
  
    public func getPrivateKey() -> Key? {
        return prvKey
    }
    
    func setPublicKey(_ key:Key) {
        self.pubKey = key
    }
  
    func setPrivateKey(_ key:Key) {
        self.prvKey = key
    }
}
